packages/react-ui-components/src/Tree/node.js

```diff
@@ -197,7 +199,10 @@ export class Header extends PureComponent {
                             style={{paddingLeft: (level * 18) + 'px'}}
                             >
                             <div className={theme.header__labelWrapper}>
-                                <IconComponent icon={icon || 'question'} label={iconLabel} className={theme.header__icon}/>
+                                {customIconComponent ?
+                                  customIconComponent :
+                                  <IconComponent icon={icon || 'question'} label={iconLabel} className={theme.header__icon} />
```

Comment:
Not the right amount of left-padding spaces, you need 4, not 2

Reply:
meh >.>

Reply:
But why 4 instead of 2 🤷‍♀️ ... anyways pushing a fix

Reply:
also: `make lint` executes the linter locally :)

Reply:
I'm not very opinionated about this but it's because of this line: https://github.com/neos/neos-ui/blob/master/.editorconfig#L7

:)

Reply:
Yeah, wondering why atom doesn't respect that setting for me